File: leetcode/0268. Missing Number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        ios_base::sync_with_stdio(false);
        cin.tie(NULL);

        const int n = nums.size();
        int ans = n * (n + 1) / 2;
        ans -= accumulate(nums.begin(), nums.end(), 0);
        return ans;
    }
};

File: leetcode/0268. Missing Number.java
class Solution {
    public int missingNumber(int[] nums) {
        final int n = nums.length;
        int ans = n * (n + 1) / 2; // arithmetic sum for 0 ~ n
        for(int num : nums){
            ans -= num;
        }
        return ans;
    }
}